Donation-platform/target/appengine-staging/WEB-INF/appengine-generated/app.yaml

```diff
@@ -0,0 +1,41 @@
+runtime: java8
```

Comment:
Why are all these target files being committed? These are just auto-generated locally; I don't think we want them in the github repo.

Reply:
I agree we might have to use .gitignore

Reply:
Please add gitignore and re-send PR, will be easier then to review. Currently it shows 244 files :)